libraries/classes/Util.php

```diff
@@ -1756,6 +1757,18 @@ public static function unsupportedDatatypes(): array
         return [];
     }
 
+    /**
+     * This function is to check whether database support UUID
+     *
+     * @return bool
+     */
+    public static function isUUIDSupported()
+    {
+        global $dbi;
+
+        return Compatibility::isUUIDSupported($dbi);
```

Comment:
We're using `$GLOBALS` instead of `global` for new code.
```suggestion
        return Compatibility::isUUIDSupported($GLOBALS['dbi']);
```

Reply:
updated. thank you. ^_^